roles/setup/tasks/main.yml

```diff
@@ -31,12 +31,14 @@
   when: receptor_install_method == 'release'
 
 - name: Check if receptor was installed correctly
+  when: receptor_install_method in ['release', 'local', 'package']
```

Comment:
All these `when` in the main tasks... I wonder if it won't be better to have task files per install method. I think we discussed this earlier @TheRealHaoLiu

Reply:
lets do this in a follow up